firefox_ui_harness/__init__.py

```diff
@@ -2,4 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from runtests import cli
+from mixins import *
```

Comment:
Mixins no longer exist. I was using them at the beginning because it was easier to get it working by mostly copying from the gaia tests. But with different entry points it was a nightmare. I will finally update the folder when all is ready.